zenoh/src/net/runtime/adminspace.rs

```diff
@@ -63,14 +69,17 @@ pub struct AdminContext {
     runtime: Runtime,
 }
 
-type Handler = Arc<dyn Fn(&AdminContext, Query) + Send + Sync>;
+type Handler = Arc<dyn Fn(&keyexpr, &AdminContext, Query) + Send + Sync>;
 
 pub struct AdminSpace {
     zid: ZenohId,
     queryable_id: QueryableId,
     primitives: Mutex<Option<Arc<Face>>>,
     mappings: Mutex<HashMap<ExprId, String>>,
-    handlers: HashMap<OwnedKeyExpr, Handler>,
+    // key is full key expression (root_key/key[/glob]), value is (handler, glob_len)
```

Comment:
I suggest you make a proper Rustdoc comment with `///`.

Reply:
In an LSP-powered editor, hovering over a reference of the variable would show you the Rustdoc.